README.md

```diff
@@ -1 +1,90 @@
-# air-pollution
\ No newline at end of file
+# Air Pollution
+> The air pollution web page offers users information on the Air Quality Index (AQI) of the world cities and the detailed measurement of each polluting substance used to measure said AQI. Users can search by city name.
+
+![screenshot 1](./ap-screenshot-1.png)
+
+![screenshot 2](./ap-screenshot-2.png)
+
+
+## Features
+
+The website displays air pollution information using react components and redux store to:
+
+- Navigation between two pages, start and contamination details.
+- Users can start a search by entering the city name in the search bar on the home page, which will display a list of matching cities.
+- Users can view the detailed information on air pollution by clicking the arrow button located in each city.
+- The details page shows the air quality index information using international standard colors and city location information.
+- The details page also shows a table with the measurement of each polluting substance.
+- The user can return to the home page by clicking the back button.
+- The website retrieves city and air pollution data from two different third-party APIs.
+
+
+## Built With
+
+- JavaScript
+- React
+- Redux
+- SCSS
+
+
+## Live Demo
+
+[World's air pollution](https://airpollutiongaby.herokuapp.com/)
+
+
+## Link loom video
+
+[Presentation](https://www.loom.com/share/de3fa4a29a424f459fa1f2ba54a36fe0)
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Have `git` installed on your local computer.
+
+### Install
+
+- Clone this repository to your local computer. Choose the location you prefer.
+- Run the command `npm install` to install all project's dependencies.
+
+### Usage
+
+- To begin the development, run the command `npm start`.
+- The project will automatically open the `index.html` file in a browser.
+
+### Deployment
+
+- To create a production bundle, run the command `npm run build`.
+- Deploy the `build` folder.
+
+
+## Authors
+
+👤 **Gabriela Sánchez Espirilla**
+
+- GitHub: [@gabyse1](https://github.com/gabyse1)
+- LinkedIn: [gabyse](https://www.linkedin.com/in/gabyse/)
+- Twitter: [@gabyse0](https://twitter.com/gabyse0)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Linters: [Microverse](https://github.com/microverseinc/linters-config)
+
```

Comment:
- [ ] Please kindly add an "Acknowledgements" section or similar that gives appropriate credit to Nelson Sakwa in the README file